### PR TITLE
Bump simple-markdown due to security changes

### PR DESCRIPTION
**Pull request by miguelslemos**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -25,7 +25,7 @@
   "dependencies": {
     "prop-types": "^15.5.10",
     "react-native-tabular-grid-markdown-view": "0.1.x",
-    "simple-markdown": "0.3.x"
+    "simple-markdown": "0.4.4"
```
Comment (parndt) [inline package.json]:
```suggestion
    "simple-markdown": "^0.4.4"
```

**Comment by parndt** [comment]
@Benjamin-Dobell hey, thanks for this library!! :smile: This is the only security advisory I see when running `npm audit`..  I appreciate you are probably busy, but wondered if you could review this and possibly #29 and release a new version of react-native-markdown-view please?

**Comment by Benjamin-Dobell** [comment]
Finally got off my bum and bumped to 0.6.1.

**Comment by pachuka** [comment]
@Benjamin-Dobell - do you mind publishing this to npm repo so we can consume the fix?

**Pull request closed without merging**